<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-KDL0RR6" ignored="0" name="TestSuite" tests="14" failures="5" timestamp="2022-01-05T13:39:15 ICT" time="106.317" errors="0">
  <testcase name="TC01" time="4.079" classname="Testcases.Railway.TC01">
    <failure type="java.lang.AssertionError" message="Welcome message is not displayed as expected expected [Welcome hehehe@mailinator.com] but found []">
      <![CDATA[java.lang.AssertionError: Welcome message is not displayed as expected expected [Welcome hehehe@mailinator.com] but found []
at Testcases.Railway.TC01.TC01(TC01.java:28)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- TC01 -->
  <testcase name="TC02" time="3.449" classname="Testcases.Railway.TC02"/>
  <testcase name="TC03" time="3.496" classname="Testcases.Railway.TC03">
    <failure type="java.lang.AssertionError" message="expected [There was a problem with your login and/or errors exist in your form.] but found [Invalid username or password. Please try again.]">
      <![CDATA[java.lang.AssertionError: expected [There was a problem with your login and/or errors exist in your form.] but found [Invalid username or password. Please try again.]
at Testcases.Railway.TC03.TC03(TC03.java:27)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- TC03 -->
  <testcase name="TC04" time="4.319" classname="Testcases.Railway.TC04"/>
  <testcase name="TC05" time="6.455" classname="Testcases.Railway.TC05">
    <failure type="java.lang.AssertionError" message="expected [You have used 4 out of 5 login attempts.After all 5 have been used, you will be unable to login for 15 minutes.] but found [Invalid username or password. Please try again.]">
      <![CDATA[java.lang.AssertionError: expected [You have used 4 out of 5 login attempts.After all 5 have been used, you will be unable to login for 15 minutes.] but found [Invalid username or password. Please try again.]
at Testcases.Railway.TC05.TC05(TC05.java:28)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- TC05 -->
  <testcase name="TC06" time="6.259" classname="Testcases.Railway.TC06"/>
  <testcase name="TC07" time="4.759" classname="Testcases.Railway.TC07">
    <failure type="java.lang.AssertionError" message="Successful register message is not displayed as expected expected [Thank you for registering your account] but found [You&amp;apos;re here]">
      <![CDATA[java.lang.AssertionError: Successful register message is not displayed as expected expected [Thank you for registering your account] but found [You're here]
at Testcases.Railway.TC07.TC07(TC07.java:29)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- TC07 -->
  <testcase name="TC09" time="6.983" classname="Testcases.Railway.TC09"/>
  <testcase name="TC10" time="3.822" classname="Testcases.Railway.TC10"/>
  <testcase name="TC11" time="3.642" classname="Testcases.Railway.TC11"/>
  <testcase name="TC14" time="7.352" classname="Testcases.Railway.TC14"/>
  <testcase name="TC15" time="5.742" classname="Testcases.Railway.TC15">
    <failure type="java.lang.AssertionError" message="expected [Ticket price from Đà Nẵng to Sài Gòn] but found [Ticket price from Sài Gòn to Phan Thiết]">
      <![CDATA[java.lang.AssertionError: expected [Ticket price from Đà Nẵng to Sài Gòn] but found [Ticket price from Sài Gòn to Phan Thiết]
at Testcases.Railway.TC15.TC15(TC15.java:39)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- TC15 -->
  <testcase name="TC16" time="9.37" classname="Testcases.Railway.TC16"/>
  <testcase name="TC17" time="9.935" classname="Testcases.Railway.TC17"/>
</testsuite> <!-- TestSuite -->
